Sources/Testing/Events/Event+FallbackHandler.swift

```diff
@@ -0,0 +1,44 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+private import _TestingInternals
+
+extension Event {
+  /// Post this event to the currently-installed fallback event handler.
+  ///
+  /// - Parameters:
+  ///   - context: The context associated with this event.
+  ///
+  /// - Returns: Whether or not the fallback event handler was invoked. If the
+  ///   currently-installed handler belongs to the testing library, returns
+  ///   `false`.
+  borrowing func postToFallbackHandler(in context: borrowing Context) -> Bool {
+    #if canImport(_TestingInterop)
+      guard let fallbackEventHandler = _swift_testing_getFallbackEventHandler() else {
+        return false
+      }
+
+      // Encode the event as JSON and pass it to the handler.
+      let encodeAndInvoke = ABI.CurrentVersion.eventHandler(encodeAsJSONLines: false) {
```

Comment:
You might want to cache this event handler somewhere as there's a bit of runtime overhead to generating it. Remember the fallback event handler can't change after its initial assignment, and will be set by the time an event is generated, so we don't need to worry about it having changed on later calls (i.e. the closure we generate here can permanently capture it.)